ethereum/contracts/Messages.sol

```diff
@@ -80,6 +80,14 @@ contract Messages is Getters {
             require(i == 0 || sig.guardianIndex > lastIndex, "signature indices must be ascending");
             lastIndex = sig.guardianIndex;
 
+            /// @dev Ensure that the provided signature index is within the
+            /// bounds of the guardianSet. This is implicitly checked by the array
```

Comment:
Nit-picky, but this comment style is not consistent with the rest of the contract.

Reply:
In what way? It looks consistent to me, insofar as it's a natspec style comment in the wrong place (the function body).

Reply:
I didn't realize natspec would interpret multiline comments using "///" instead of "/**"

Reply:
we should probably remove the natspecs from inside the function bodies, as I don't think they actually get parsed, but that's outside of the scope of this PR